src/main/webapp/app/stores/curation-page.store.ts

```diff
@@ -133,6 +137,10 @@ export class CurationPageStore {
       .filter(query => query.queryId && this.annotatedAltsCache.cache[query.queryId] && this.annotatedAltsCache.cache[query.queryId].result)
       .map(query => (query.queryId !== undefined ? this.annotatedAltsCache.cache[query.queryId].result : undefined));
   }
+
+  setReadOnly(isAnotherUserReviewing: boolean) {
+    this.readOnly = isAnotherUserReviewing ? true : false;
```

Comment:
Already a boolean value, can just set as is.
```suggestion
    this.readOnly = isAnotherUserReviewing;
```